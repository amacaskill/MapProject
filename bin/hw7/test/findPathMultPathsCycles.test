# tests for graphs with paths that include cycles

CreateGraph g 

AddNode g n1
AddNode g n2
AddNode g n3
AddNode g n4

AddEdge g n1 n2 1 
AddEdge g n2 n3 1 
AddEdge g n3 n4 1
AddEdge g n1 n3 3
AddEdge g n2 n1 0.1
AddEdge g n3 n1 0.1
AddEdge g n3 n2 0.1

FindPath g n1 n4