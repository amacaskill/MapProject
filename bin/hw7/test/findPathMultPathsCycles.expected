# tests for graphs with paths that include cycles

created graph g 

added node n1 to g
added node n2 to g
added node n3 to g
added node n4 to g

added edge 1.000 from n1 to n2 in g
added edge 1.000 from n2 to n3 in g
added edge 1.000 from n3 to n4 in g
added edge 3.000 from n1 to n3 in g
added edge 0.100 from n2 to n1 in g
added edge 0.100 from n3 to n1 in g
added edge 0.100 from n3 to n2 in g

path from n1 to n4:
n1 to n2 with weight 1.000
n2 to n3 with weight 1.000
n3 to n4 with weight 1.000
total cost: 3.000